services/exam-manager/app/main.py

```diff
@@ -83,12 +81,12 @@ async def readiness() -> dict:
         500: Any of the exam-tree tables does not exist
     """
     ins = inspect(engine)
-    exam_tables = ["exam", "procedure", "job", "record"]
-    print(exam_tables)
-    print(ins.get_table_names())
-    # if not all(t in exam_tables for t in ins.get_table_names()):
+    existing_tables = ins.get_table_names()
+    required_tables = ["exam", "procedure", "job", "record"]
+    print("Existing tables: ", existing_tables)
 
-    #     raise HTTPException(status_code=500, detail="SQL-DB: Could not create all required tables.")
+    if not all(t in existing_tables for t in required_tables):
```

Comment:
nested and hard to read for me - is t even needed?

Reply:
This statement simply checks if each entry of `required_tables` is contained `existing_tables`. It is only true if this is the case for `all()` entries. Otherwise the exception is raised. I would consider this as "pythonic".